### PR TITLE
PXB-2698 Investigate and fix PDF publishing (2.4)

### PR DESCRIPTION
**Pull request by patrickbirch**
M source/advanced/encrypted_innodb_tablespace_backups.rst
 M source/glossary.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/glossary.rst [hunk 1]
```diff
@@ -4,136 +4,199 @@
...
-.opt
-  *MySQL* stores options of a database (like charset) in a file with an `.opt` extension in the database directory.
+   .par
+     Each partitioned table has .par file which contains metadata about the
+     partitions.
 
-.par
-  Each partitioned table has .par file which contains metadata about the partitions.
+   .TRG
+     File containing the Triggers associated to a table, for example
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/glossary.rst]: File containing the Triggers associated with a table, for example,

File: storage/innobase/xtrabackup/doc/source/glossary.rst [hunk 2]
```diff
@@ -4,136 +4,199 @@
...
+     `:file:`mytable.TRG`. With the :term:`.TRN` file, they represent all the
+     Trigger definitions.
 
-.TRG
-  File containing the Triggers associated to a table, for example,
-  ``mytable.TRG`. With the `.TRN` file, they represent all the
-  Trigger definitions.
+   .TRN
+     File containing the Triggers' Names associated to a table, for example
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/glossary.rst]: File containing the Triggers' Names associated with a table, for example,

File: storage/innobase/xtrabackup/doc/source/glossary.rst [hunk 3]
```diff
@@ -4,136 +4,199 @@
...
-   Default prefix for tablespace files, for example, `ibdata1` is a 10MB
-   auto-extensible file that *MySQL* creates for the shared tablespace by
-   default.
+   configuration file
+      The file that contains the server startup options. 
 
-InnoDB
-  Storage engine which provides ACID-compliant transactions and foreign key support, among others improvements over `MyISAM`. It is the default engine for *MySQL* as of the 5.5 series.
+   crash
+      An unexpected shutdown which does not allow the normal server shutdown cleanup activities.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/glossary.rst]: An unexpected shutdown that

File: storage/innobase/xtrabackup/doc/source/glossary.rst [hunk 4]
```diff
@@ -4,136 +4,199 @@
...
 
-    [mysqld]
-    innodb_buffer_pool_size=8MB
+   full backup
+     A backup that contains the complete source data from an instance. 
 
-innodb_data_file_path
-  Specifies the names, sizes and location of shared tablespace files:
+   ibdata
+     Default prefix for tablespace files, for example :file:`ibdata1` is a 10MB
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/glossary.rst]: files, for example,

File: storage/innobase/xtrabackup/doc/source/glossary.rst [hunk 5]
```diff
@@ -4,136 +4,199 @@
...
+     LSN. The LSN is the system version number for the entire database. Each
+     page's LSN shows how recently it was changed.
+
+   my.cnf
+     The database server's main configuration file. Most
+     Linux distributions place it as :file:`/etc/mysql/my.cnf` or
+     :file:`/etc/my.cnf`, but the location and name depends on the particular
+     installation. Note that this is not the only way of configuring the
+     server, some systems does not have one and rely on the command
+     options to start the server and its defaults values.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/glossary.rst]: default values.

File: storage/innobase/xtrabackup/doc/source/glossary.rst [hunk 6]
```diff
@@ -4,136 +4,199 @@
...
+     Previous default storage engine for |MySQL| for versions prior to 5.5. It
+     doesn't fully support transactions but in some scenarios may be faster
+     than :term:`InnoDB`. Each table is stored on disk in 3 files:
+     :term:`.frm`, :term:`.MYD`, :term:`.MYI`.
+  
+   physical backup
+     A backup that copies the data files.
+
+   point in time recovery
+     This method allows data to be restored into the state it was in any selected point of time.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/glossary.rst]: This method allows data to be restored to the condition it was at any selected point in time.

File: storage/innobase/xtrabackup/doc/source/glossary.rst [hunk 7]
```diff
@@ -4,136 +4,199 @@
...
+     Copies the database backups taken using the backup command to the original location or a different location. A restore returns data that has been either lost, corrupted, or stolen to the original condition at a specific point in time.
+ 
+   xbcrypt
+     To support encryption and decryption of the backups, a new tool xbcrypt
+     was introduced to |Percona XtraBackup|. This utility has been modeled
+     after the xbstream binary to perform encryption and decryption outside of
+     |Percona XtraBackup|.
+
+   xbstream
+     To support simultaneous compression and streaming, |Percona XtraBackup| uses the xbstream format. For more information see :option:`--stream`
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/glossary.rst]: For more information, see

File: storage/innobase/xtrabackup/doc/source/glossary.rst [hunk 8]
```diff
@@ -4,136 +4,199 @@
...
+   .CSV
+     Each table with the :program:`CSV Storage` engine has ``.CSV`` file which
+     contains the data of it (which is a standard Comma Separated Value file).
 
-.CSV
-  Each table with the `CSV Storage` engine has ``.CSV`` file which contains the data of it (which is a standard Comma Separated Value file).
+   .exp
+     Files with the ``.exp`` extension are created by |Percona XtraBackup| per
+     each |InnoDB| tablespace when the :option:`--export` option is
+     used on prepare. See :doc:`restoring individual tables <xtrabackup_bin/restoring_individual_tables>`.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/glossary.rst]: in preparation

File: storage/innobase/xtrabackup/doc/source/glossary.rst [hunk 9]
```diff
@@ -4,136 +4,199 @@
...
-  Files with the ``.exp`` extension are created by *Percona XtraBackup* per each *InnoDB* tablespace when the `xtrabackup --export` option is used on prepare. These files can be used to import those tablespaces on *Percona Server for MySQL* 5.5 or lower versions, see :doc:`restoring individual tables <xtrabackup_bin/restoring_individual_tables>`".
+   .frm
+     For each table, the server will create a file with the ``.frm`` extension
+     containing the table definition (for all storage engines).
 
-.frm
-  For each table, the server creates a file with the ``.frm`` extension which contains the table definition (for all storage engines).
+   .ibd
+     On a multiple tablespace setup (:term:`innodb_file_per_table` enabled),
+     |MySQL| will store each newly created table on a file with a ``.ibd``
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/glossary.rst]: in a file

File: storage/innobase/xtrabackup/doc/source/glossary.rst (hunk 8 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/glossary.rst]: Updated.

File: storage/innobase/xtrabackup/doc/source/glossary.rst (hunk 9 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/glossary.rst]: Updated.

File: storage/innobase/xtrabackup/doc/source/glossary.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/glossary.rst]: updated - also redid the random capitalization.

File: storage/innobase/xtrabackup/doc/source/glossary.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/glossary.rst]: Rewritten.

File: storage/innobase/xtrabackup/doc/source/glossary.rst (hunk 3 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/glossary.rst]: updated.

File: storage/innobase/xtrabackup/doc/source/glossary.rst (hunk 4 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/glossary.rst]: Added comma

File: storage/innobase/xtrabackup/doc/source/glossary.rst (hunk 5 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/glossary.rst]: Removed extra "s"

File: storage/innobase/xtrabackup/doc/source/glossary.rst (hunk 6 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/glossary.rst]: Condition may have other meanings.

File: storage/innobase/xtrabackup/doc/source/glossary.rst (hunk 7 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/glossary.rst]: Comma added.

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**